packages/server/src/process_move_service.ts

```diff
@@ -0,0 +1,97 @@
+import { Board, Marker, Winner } from "@mapistry/take-home-challenge-shared";
+import { WinningLines } from "./constants";
+
+export class ProcessMoveService {
+    board: Board;
+
+    constructor(board: Board) {
+        this.board = board;
+    }
+
+    static calculateMinimaxScore(board: Board, depth: number, aiTurn: boolean) {
```

Comment:
There are a few static functions in here and I'm not sure how idiomatic that is, definitely a spot I'd love some feedback. I went this direction because `calculateMinimaxScore` needed its own copy of board (not the original passed into the class) and the compiler didn't like that the function didn't reference `this`. I would think that extracting these static methods into a new class that could use instance functions would be a good step.